### ISSUE TITLE
'size' and 'totalPages' don't update when 'totalItems===0'

### ISSUE THREAD

**Issue by Nill-Aarstiderne** [lucasmogari/react-pagination#6]
I have a component where I can filter the items.
If the filter yields 0 results, the usePagination-hook returns old values for`size` and `totalPages`.

E.g. I start with 70 items and show 10 items on each page.
The result is `totalPages: 7`.

If I filter to show 40 items, `totalPages` updates to 4.
However, if I then filter to show 0 items, `totalPages` sticks to the old value and the result is still 4.

The issues seems to be in the `paginationReducer`, where `totalItems` is not updated when `action.totalItems` is 0:
```
      if (action.totalItems) {
        totalItems = Number(action.totalItems);
      }
```

**Comment by lucasmogari** [comment]
Hi. Thanks for opening this issue. The problems is fixed in version 1.1.4.

**Comment by Nill-Aarstiderne** [comment]
Hi Lucas,

I want to express my sincere gratitude for your quick response and fix – I genuinely appreciate your work on this package. It's been an incredible learning experience for me as I'm relatively new to React, and your library has provided invaluable insights into the power of hooks.

On a side note, I noticed a small change in the npm package between versions 1.1.3 and 1.1.4. In 1.1.3, the `\dist\react-pagination.es.js` file is in a readable format, while in 1.1.4, it has been minified. This doesn't affect the overall functionality, but it might make it a bit more challenging to debug potential issues in the future. Not that I've encountered many issues with your code – it's quite solid! I just thought it was worth mentioning.

Once again, thank you for your fantastic work and your quick response to the bug I reported.
